vita3k/gui/src/gui.cpp

```diff
@@ -32,10 +32,12 @@
 #include <lang/functions.h>
 #include <packages/sfo.h>
 #include <regmgr/functions.h>
+#include <touch/functions.h>
 #include <util/fs.h>
 #include <util/log.h>
 #include <util/string_utils.h>
 
+#include <SDL_events.h>
```

Comment:
Is there a reason for this include?

Reply:
Before yess lol, but now i have totaly refactor code for this, so no need anymore